core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -305,6 +307,15 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }.map(RequestComplete(_))
   }
 
+  def unregisterBillingProject(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = {
+    for {
+      _ <- dataSource.inTransaction { dataAccess => dataAccess.rawlsBillingProjectQuery.delete(projectName) }
+      _ <- samDAO.deleteResource(SamResourceTypeNames.billingProject, projectName.value, userInfo)
```

Comment:
@helgridly did you want this to call sam too? This will only work in fiab.

Reply:
Yes, register also calls Sam. "Only work in fiab" as opposed to? Locally?

Reply:
only fiabs will have the version of the billing-project config that has the "delete" action, so this call will fail in any other environment

so GPAlloc can either call the sam endpoint itself and i'll remove this line or i can keep this call here and fail gracefully. i don't really care which

Reply:
I'd rather leave it here and fail as (dis)gracefully as you please. It should be an obvious failure mode at least